cppcache/src/TcrMessage.cpp

```diff
@@ -1015,6 +1015,36 @@ void TcrMessage::processChunk(const std::vector<uint8_t>& chunk, int32_t len,
         break;
       }
       // fall-through for other cases
+      if (m_chunkedResult != nullptr) {
```

Comment:
What's this block of code doing?  I don't immediately see a place where it was removed/moved, so is it new?

Reply:
The previous code did a fallthrough if it didn't make it to the `break;` statement on line 999 or 1015 (which is buried in an `if` and `if else`).  This is the same code as on line 1054.  I thought about extracting a function but got lazy